.circleci/config.yml

```diff
@@ -639,15 +632,12 @@ workflows:
             - functional_tests_38_grype
             - integration_tests_38
             - test_container_rebuild
-      - anchore/image_scan:
-          name: anchore_image_scan
+      - grype/list-image-vulns:
           context: dockerhub
           requires:
             - push_dev_image
-          image_name: anchore/anchore-engine-dev:${CIRCLE_SHA1}
-          private_registry: true
-          timeout: "2000"
-          policy_failure: true
+          image-name: anchore/anchore-engine-dev:${CIRCLE_SHA1}
```

Comment:
This can use ${CIRCLE_TAG} for the image tag.

Reply:
updated. thanks!